src/AppInstallerCLICore/Command.cpp

```diff
@@ -42,8 +43,35 @@ namespace AppInstaller::CLI
 
     void Command::OutputIntroHeader(Execution::Reporter& reporter) const
     {
+        auto infoOut = reporter.Info();
+        VirtualTerminal::ConstructedSequence indent;
+
+        if (reporter.SixelsEnabled())

```

Comment:
Do we want to try CATCH_LOG since I saw a bunch of THROW_IF_FAILED in sixels.cpp code?